src/api/app/models/review.rb

```diff
@@ -54,6 +61,16 @@ class NotFoundError < APIError
   before_validation :set_reviewable_association
   after_commit :update_cache
 
+  def review_assignment
+    # NOTE: they can disappear later and the review should be still
```

Comment:
This comment is now cut. It used to also have these 2 additional lines:

```
# usable to some degree (can be showed at least)
# But it must not be possible to create one with broken references
```

Reply:
Hm, not sure why I added the comment again... Dropped it now.

IMO there is no reason or need to allow to remove by_* attributes after a review was created. They are duplications of the object references (user_id, group_id, ...) and should in my opinion stay. Even when the referenced object got deleted.
Otherwise I don't see any reason to have these non-relational references in first place.